oak_functions_freestanding/src/remote_attestation.rs

```diff
@@ -44,95 +36,48 @@ pub struct PublicKeyInfo {
     pub attestation: Vec<u8>,
 }
 
+pub trait Handler {
+    fn handle(&mut self, request: &[u8]) -> anyhow::Result<Vec<u8>>;
+}
+
 pub trait AttestationHandler {
-    fn message(&mut self, session_id: SessionId, body: &[u8]) -> anyhow::Result<Vec<u8>>;
+    fn message(&mut self, body: &[u8]) -> anyhow::Result<Vec<u8>>;
     fn get_public_key_info(&self) -> PublicKeyInfo;
 }
 
-pub struct AttestationSessionHandler<F, G: AttestationGenerator, V: AttestationVerifier> {
-    session_tracker: SessionTracker<G, V>,
-    request_handler: F,
-    public_key_info: PublicKeyInfo,
+pub struct AttestationSessionHandler<H: Handler> {
+    // TODO(#3442): Use attestation generator to attest public key.
```

Comment:
super nit: I think "attest to the public key", or "provide attestation evidence for the public key" is more accurate.

Reply:
Done.